README.md

```diff
@@ -730,8 +730,12 @@ OR
   * `./ccd enable backend message-publisher`
   * Run docker-compose `./ccd compose up -d`
   * Verify that ccd-message-publisher is up and running by `curl localhost:4456/health`
-
-
+ 
+* To enable **ccd-case-document-am-api**
+  * `./ccd enable backend frontend dm-store`
+  * run docker-compose `./ccd compose up -d`
+  * verify that ccd-case-document-am-api is up and running by `curl localhost:4455/health`
+    
```

Comment:
If AC highlighted in your [previous comment](https://github.com/hmcts/ccd-docker/pull/119/files#r591422433) requires `ccd-case-document-am-api` to be enabled by default then this text is not required and it should be aded to the Back-end list of services started on line 656, i.e. [here](https://github.com/hmcts/ccd-docker/tree/RDM-11416-Case_Document_Docker#enabling-additional-projects)).

However it may be worth mentioning this AC  on the team channel for re-review (or for discussion on our next team call).  i.e. 
* are we now expanding our default to include the all doc-store items, (case-docs, dm-store, azure-storage etc).
* or will case-docs point to a stub
* or will we use a big `ES_ENABLED_DOCKER` / `ELASTIC_SEARCH_ENABLED` like switch that disables doc-store docker integration and FTA tests when people are running without it?

Reply:
Fixed in https://github.com/hmcts/ccd-docker/pull/119/commits/013bf7fcf6ecdbed6b3910dfc483196caf41b9c4

`ccd-case-document-am-api` is now an optional component in its own compose file that needs to be explicitly enabled

Reply:
@sullivangareth, Is the new compose file missing from this PR?